app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class RecipesController < ApplicationController
+  def show
+    @recipes = Recipe.find(params[:id])
```

Comment:
This is fine, your variable probably should be singular since this is referencing one specific Recipe instance.